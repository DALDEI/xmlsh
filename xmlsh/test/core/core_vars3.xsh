# core_vars3.xsh
# Test multi var prefixes

a=b eval 'echo a=$a'
b= xtype -v b
a=b c=d e= eval 'echo a:$a c:$c e:$e'

