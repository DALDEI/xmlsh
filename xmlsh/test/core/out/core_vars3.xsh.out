a=b
empty-sequence()
a:b c:d e:
